inst/www/srcjs/shinyalert.js

```diff
@@ -13,15 +12,27 @@ Shiny.addCustomMessageHandler('shinyalert.show', function(params) {
   }
 
   var callbackR = function(value) {};
-  if (params['cbid'] != null) {
-    var cbid = params['cbid'];
-    delete params['cbid'];
+  // Always create cbid
+  var cbid = params['cbid'];
+  delete params['cbid'];
+  if(typeof cbid === 'string'){
```

Comment:
Is cbid not **alwasy** going to be a string?

Reply:
We can remove the `if` statement if you always include `cbid` in R code.

Reply:
It wasn't before, but my understanding is that your PR ensures that `cbid` is ALWAYS going to be in the javascript parameters, in which case this check is redundant. Is that right?